### PR TITLE
Use monitoring from aos_common

### PR DESCRIPTION
**Pull request by MykolaSolyanko**

### CODE REVIEW

File: launcher/launcher_test.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ import (
 	"github.com/aoscloud/aos_servicemanager/config"
 	"github.com/aoscloud/aos_servicemanager/launcher"
 	"github.com/aoscloud/aos_servicemanager/layermanager"
-	"github.com/aoscloud/aos_servicemanager/monitoring"
```
Comment (Eabramov84) [inline launcher/launcher_test.go:49]: import should be public part

File: monitoringcontroller/cover.out [hunk 2]
```diff
@@ -0,0 +1,4 @@
+mode: set
```
Comment (Eabramov84) [inline monitoringcontroller/cover.out:1]: remove this file

File: monitoringcontroller/monitoringcontroller.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package monitoringcontroller
+
+import (
+	"github.com/aoscloud/aos_communicationmanager/cloudprotocol"
```
Comment (Eabramov84) [inline monitoringcontroller/monitoringcontroller.go:21]: use common for cloudeprotocal

File: monitoringcontroller/monitoringcontroller_test.go [hunk 4]
```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2021 Renesas Electronics Corporation.
```
Comment (Eabramov84) [inline monitoringcontroller/monitoringcontroller_test.go:3]: 2022 year

File: monitoringcontroller/monitoringcontroller_test.go [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+package monitoringcontroller_test
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/aoscloud/aos_common/monitoring"
+	"github.com/aoscloud/aos_communicationmanager/cloudprotocol"
```
Comment (Eabramov84) [inline monitoringcontroller/monitoringcontroller_test.go:26]: use common for cloudprotocol

File: config/config.go [hunk 6]
```diff
@@ -44,14 +45,6 @@ const (
  * Types
  ******************************************************************************/
```
Comment (Eabramov84) [inline config/config.go:46]: make header length 120

File: launcher/launcher.go [hunk 7]
```diff
@@ -37,14 +37,14 @@ import (
 	"github.com/aoscloud/aos_common/api/cloudprotocol"
 	"github.com/aoscloud/aos_common/utils/action"
 	"github.com/aoscloud/aos_common/utils/fs"
+	"github.com/aoscloud/aos_common/monitoring"
```
Comment (al1img) [inline launcher/launcher.go:40]: Commit message: use monitoring from aos_common?

**Review by al1img — COMMENTED** [review]
where are changes in servicemanager.go?

File: monitoringcontroller/cover.out (hunk 2 above)
Comment (MykolaSolyanko) [reply monitoringcontroller/cover.out:1]: Done

File: monitoringcontroller/monitoringcontroller.go (hunk 3 above)
Comment (MykolaSolyanko) [reply monitoringcontroller/monitoringcontroller.go:21]: Done

File: monitoringcontroller/monitoringcontroller_test.go (hunk 4 above)
Comment (MykolaSolyanko) [reply monitoringcontroller/monitoringcontroller_test.go:3]: Done

File: monitoringcontroller/monitoringcontroller_test.go (hunk 5 above)
Comment (MykolaSolyanko) [reply monitoringcontroller/monitoringcontroller_test.go:26]: Done

**Comment by MykolaSolyanko** [comment]
> where are changes in servicemanager.go?

Will be done in separate PR

File: launcher/launcher_test.go (hunk 1 above)
Comment (MykolaSolyanko) [reply launcher/launcher_test.go:49]: Done

File: launcher/launcher.go (hunk 7 above)
Comment (MykolaSolyanko) [reply launcher/launcher.go]: Done

File: monitoringcontroller/monitoringcontroller.go [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+package monitoringcontroller
+
+import (
+	"github.com/aoscloud/aos_common/api/cloudprotocol"
+	"github.com/aoscloud/aos_servicemanager/config"
+	log "github.com/sirupsen/logrus"
+)
+
+// MonitorController instance.
```
Comment (al1img) [inline monitoringcontroller/monitoringcontroller.go:26]: Should be under `Types` header below `Consts`

File: monitoringcontroller/monitoringcontroller.go [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+type MonitorController struct {
```
Comment (al1img) [inline monitoringcontroller/monitoringcontroller.go:27]: `monitoringcontroller.MonitorController` looks not so good. Consider to rename to just `Controller` or `Instance`.

File: monitoringcontroller/monitoringcontroller.go (hunk 8 above)
Comment (MykolaSolyanko) [reply monitoringcontroller/monitoringcontroller.go:26]: Done

File: monitoringcontroller/monitoringcontroller.go (hunk 9 above)
Comment (MykolaSolyanko) [reply monitoringcontroller/monitoringcontroller.go:27]: chose `Controller`

File: monitoringcontroller/monitoringcontroller.go [hunk 10]
```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: Apache-2.0
```
Comment (al1img) [inline monitoringcontroller/monitoringcontroller.go:1]: Package name `monitorcontroller`?

Comment (MykolaSolyanko) [reply monitoringcontroller/monitoringcontroller.go]: Done

**Review by al1img — APPROVED** [review]

**Pull request merged**